File: Backendd/ANIMO_Backend/src/main/java/edu/isep/animo_backend/models/Address.java
package edu.isep.animo_backend.models;

import jakarta.persistence.*;
import lombok.Getter;
import lombok.Setter;

@Entity
@Table(name = "tblAddresses")
public class Address {


    @Id
    @GeneratedValue(strategy = GenerationType.AUTO)
    private @Getter @Setter Long id;
    private @Getter @Setter String street;
    private @Getter @Setter String number;
    private @Getter @Setter String zipCode;
    private @Getter @Setter String city;
    private @Getter @Setter String country;


}

File: Frontend/ANIMO/src/app/modules/shared/models/address.ts
export class Address {
    id: number;
    street: string;
    number: string;
    zipCode: string;
    city: string;
    country: string;
  
    constructor(id: number, street: string, number: string, zipCode: string, city: string, country: string) {
      this.id = id;
      this.street = street;
      this.number = number;
      this.zipCode = zipCode;
      this.city = city;
        this.country = country;
    }
  }
